site/src/pages/guides/adding-a-second-entrypoint.mdx

```diff
@@ -0,0 +1,110 @@
+# Adding a second entrypoint
+
+When adding a second entrypoint, you are creating a second place where a package can be accessed from. Note that much of the time you do not need a second entrypoint. We have a guide on [when to use multiple entrypoints](/guides/when-should-i-use-multiple-entrypoints).
+
+So, let's assume we have a package called `emoji-uid` that gets us three random emojis, but we want to make a second function that only uses the best emojis. What we want to be able to do is write is:
+
+```js
+import getEmojiId from "emoji-uid";
+import getAwesomeId from "emoji-uid/awesome";
+```
+
+With `preconstruct`'s base setup, this isn't possible, so we will need to add a new `awesome` entrypoint.
+
+First, our project structure looks like:
+
+```
+- src
+    /index.js
+- package.json
+```
+
+and our package.json looks like:
+
+```json
+{
+  "name": "emoji-uid",
+  "version": "1.0.2",
+  "description": "Get 🔥 UIDs to help make things 💯",
+  "main": "dist/emoji-uid/.cjs.js",
+  "preconstruct": {
+    "source": "src/index.js"
+  }
+}
+```
+
+Let's assume we've written our new function and written it to `src/awesomenewFunction.js`
+
+The first bit of config we need to do is to add an `entrypoints` field to our preconstruct config. We should modify this in our package.json to read:
+
+```json
+  "preconstruct": {
+    "source": "src/index.js"
```

Comment:
```suggestion
    "source": "src/index.js",
```

Reply:
Mostly wanted to have an existing `preconstruct` field to add things to as a pointer to where to add things. Can make it something other than `src/index.js`.

Reply:
I don't mind - Happy to leave it how it is.